content/api-streamer/overview.textile

```diff
@@ -0,0 +1,67 @@
+---
+title: Overview
+section: api-streamer
+index: 65
+---
+
+h2(#what-is-api-streamer). What is API Streamer?
+
+"API Streamer":https://www.ably.io/api-streamer makes it easy for developers to publish realtime APIs (also known as event-driven APIs or streaming APIs). Built on <a href="https://www.ably.io/dsn">Ably's global Data Stream Network</a>, API Streamer provides an additional layer of tooling and features for deploying streaming data to the Ably network, managing data streams, and distributing those data streams to developer teams at third party organizations through realtime push and pull subscription APIs.
+
+<a href="https://www.ably.io/api-streamer">
+  <img src="/images/diagrams/api-streamer-overview.png" style="width: 100%" alt="Ably API Streamer">
+</a>
+
+h3(#why-use-api-streamer). Why use the API Streamer to build realtime APIs for your streaming data?
```

Comment:
I think it's less a `why`, more of a `what` are the use cases?

Reply:
Up for debate, will put down in the issue to discuss later.